lms/static/sass/base/_variables.scss

```diff
@@ -187,6 +187,7 @@ $m-blue-t0: rgba($m-blue,0.125);
 $m-blue-t1: rgba($m-blue,0.25);
 $m-blue-t2: rgba($m-blue,0.50);
 $m-blue-t3: rgba($m-blue,0.75);
+$m-blue-hover: rgb(223,242,247); // #DFF2F7
```

Comment:
we should rely on an existing blue here instead of a separate hover blue if we can. Additionally, it would be helpful to call this out by separating it with a comment #used in learner profile styling (so that we can more easily clean up our blue color usage later)